src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -124,17 +128,17 @@ internal sealed class AggregatorStore
 
         if (this.OutputDeltaWithUnusedMetricPointReclaimEnabled)
```

Comment:
Since we are always going to reserve two MetricPoints now we should change the code above from https://github.com/open-telemetry/opentelemetry-dotnet/blob/bae8a6b6991b5072c1b6d8412e329257cdfca953/src/OpenTelemetry/Metrics/AggregatorStore.cs#L113-L121 to

```csharp
int reservedMetricPointsCount = 2;

// Setting metricPointIndex to 1 as we would reserve the metricPoints[1] for overflow attribute.
// Newer attributes should be added starting at the index: 2
this.metricPointIndex = 1;
```